SimTracker/SiPhase2Digitizer/plugins/Pixel3DDigitizerAlgorithm.h

```diff
@@ -55,13 +55,15 @@ class Pixel3DDigitizerAlgorithm : public Phase2TrackerDigitizerAlgorithm {
                      const std::vector<DigitizerUtility::SignalPoint>& collection_points) override;
 
 private:
-  // Raidus of Column np and ohmic
-  float _np_column_radius;
-  float _ohm_column_radius;
+  // Radius of Column np and ohmic
+  const float _np_column_radius;
```

Comment:
Code rule 2.14: '_' should not be used as first character http://cms-sw.github.io/cms_coding_rules.html
though this usage precedes this PR, it would be good to correct it here (for both the functions and the variables)